RELEASE-PLAN.md

```diff
@@ -0,0 +1,53 @@
+# Background and Context
+
+Releases of the Pathfinder Framework (Version 2 can be found [here](https://www.carbon-transparency.com/media/jpslsujn/pathfinder-framework.pdf)) are strongly
+informing the tech specs releases.
+
+Therefore, this timeline page contains the release plan for both, the Pathfinder Framework and the Pathfinder Tech Specs.
+
+In addition, the tables below disclose deprecation of releases and subsequent sunset dates.
+
+## Definitions
```

Comment:
@bhadley please review specifically the definitions here

Reply:
Looks good to me, very clear and succinct. For consistency also updated the same definitions in the Notion page to match these.